db/seeds.rb

```diff
@@ -8,9 +8,21 @@
     {
         name: 'Fluff',
         age: 6, 
-        enjoys: 'Nice Pets',
+        enjoys: 'Nice Pets and toys.',
         image: 'https://www.instagram.com/p/CbgImoUlkg2pq1xqyCgYKbeSnvHTyJ5wX-BW940/' 
+    },
+    {
+        name: 'Jam',
+        age: 4, 
+        enjoys: 'Hugs and Catnip!',
+        image: 'https://i.ytimg.com/vi/nMMPCVgD-BQ/maxresdefault.jpg' 
+    }, 
+    {
+        name: 'Mike',
+        age: 8, 
+        enjoys: 'Food' 
```

Comment:
Mike seems to be missing an image.  This will likely cause problems in the future when running seeds.